sleap/gui/commands.py

```diff
@@ -2984,6 +2985,7 @@ def set_visible_nodes(
         copy_instance: Optional[Union[Instance, PredictedInstance]],
         new_instance: Instance,
         mark_complete: bool,
```

Comment:
We might also want to pass in the `init_method` into `AddInstance.set_visible_nodes` and use `init_method == "best"` in addition to `location is not None` to determine whether the offset should be non-zero.